src/diffusers/__init__.py

```diff
@@ -160,6 +160,15 @@
             "apply_pyramid_attention_broadcast",
         ]
     )
+    _import_structure["image_processor"] = [
+        "IPAdapterMaskProcessor",
+        "PixArtImageProcessor",
+        "VaeImageProcessor",
+        "VaeImageProcessorLDM3D",
+    ]
+    _import_structure["video_processor"] = [
+        "VideoProcessor",
+    ]
```

Comment:
These changes seem unrelated?

Reply:
It's unrelated, but if it doesn't hurt then it might be more convenient to include it so we can preview the docs locally. Let me know what you think! :)